pymeasure/instruments/attocube/anc300.py

```diff
@@ -132,14 +138,26 @@ class Axis(Channel):
     stepu = Instrument.setting(
         "stepu %d",
         """Set the steps upwards for N steps. Mode must be 'stp' and N must be
-        positive.""",
-        validator=strict_range, values=[0, inf], check_set_errors=True)
+        positive. 0 causes a continous movement until stop is called.
+
+        .. deprecated:: 0.13.0 Use attr:`continuous_move` or meth:`move_raw` instead.
```

Comment:
Is `continuous_move` still present? (Similarly below)
```suggestion
        .. deprecated:: 0.13.0 Use meth:`move_raw` instead.
```